howso/utilities/features.py

```diff
@@ -158,9 +158,9 @@ def serialize(  # noqa: C901
     @classmethod
     def deserialize(
         cls,
-        data: Iterable[Iterable[Any]] | Iterable[Mapping[str, Any]],
+        data: Iterable[Iterable[t.Any]] | Iterable[Mapping[str, t.Any]],
```

Comment:
I think I'd do: `data: Iterable[Iterable[t.Any] | Mapping[str, t.Any]],` instead.

Reply:
Also, similar to the previous method, we should document when columns must be provided and we should document the raised exceptions.

Reply:
I don't see any raised exceptions for the deserialized function

Reply:
True.